Car_Localizer/Car_Localizer/Base_loss_model/loss_model_first.py

```diff
@@ -0,0 +1,20 @@
+from re import T
+from Base_loss_model.loss_model_base import Loss_Base, TensorType, ShapeType
+
+import torch
+import torch.nn as nn
+import numpy as np
+
+
+class Loss_First(Loss_Base):
+    def __init__(
+        self,
+        name: str = "Loss_First",
+        input_shape: ShapeType = (1, 3, 64, 64),
```

Comment:
input shape and num classes are not needed in both Base and Loss First (It should be called `Loss_MSE`)